includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -687,23 +689,14 @@ private function refresh_profile_data() {
 	 * filter.
 	 *
 	 * @since 1.0.0
+	 * @deprecated n.e.x.t
 	 *
 	 * @return bool True if proxy authentication is used, false otherwise.
 	 */
 	public function using_proxy() {
-		$credentials = $this->get_client_credentials();
-
-		// If no credentials yet, assume true.
-		if ( ! is_object( $credentials ) || empty( $credentials->web->client_id ) ) {
-			return true;
-		}
-
-		// If proxy credentials, return true.
-		if ( false !== strpos( $credentials->web->client_id, '.apps.sitekit.withgoogle.com' ) ) {
-			return true;
-		}
+		_deprecated_function( __METHOD__, 'n.e.x.t', 'Credentials::using_proxy()' );
```

Comment:
Let's update the replacement function to use the equivalent of `__METHOD__` for that method.
```suggestion
		_deprecated_function( __METHOD__, 'n.e.x.t', Credentials::class . '::using_proxy' );
```

Reply:
Fixed